src/dune_rules/dune_rules.ml

```diff
@@ -17,6 +17,7 @@ module Modules = Modules
 module Module_compilation = Module_compilation
 module Exe_rules = Exe_rules
 module Lib_rules = Lib_rules
+module Jsoo_rules = Jsoo_rules
```

Comment:
Why is it necessary to export  this?

Reply:
For the expect test requested by @Alizter